src/ui/pages/BridgeForm/QuoteList/QuoteList.tsx

```diff
@@ -206,6 +209,11 @@ export function QuoteList({
                   </label>
                 );
               })}
+              {isLoading ? (
+                <div style={{ height: 32 }}>
```

Comment:
Added fixed height to avoid scroll blinking (when rotating, spinner slightly changes height which impacts `fit-content` dialog)